judo-runtime-core-jsl-itest/common/src/main/java/hu/blackbelt/judo/runtime/core/jsl/fixture/JudoRuntimeExtension.java

```diff
@@ -0,0 +1,31 @@
+package hu.blackbelt.judo.runtime.core.jsl.fixture;
```

Comment:
This extension is JudoRuntimeExtensionByClass - as in dataSource. The extension name somehow must contain what lifecycle mapped for

Reply:
BeforeEcahCallback and AfterEachCallack is not used :) As a matter of fact it ios a SingletonExtension, because it is provides a single runtimeFixture for the whole class. Maybe the fixture creation can be in a BeforeAllCallback implementation meyhod.

Reply:
I made a new Extension that contains JudoDatasourceFixture and JudoRuntimeFixture and contains all of the Callbacks